inc/integrations/class-fb-instant-articles.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * WP Irving integration for FB Instant Articles.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving\Integrations;
+
+use WP_Irving\Singleton;
+use WP_Irving\Components;
+
+/**
+ * Facebook Instant Articles.
+ */
+class FB_Instant_Articles {
+	use Singleton;
+
+	/**
+	 * Constructor for class.
+	 */
+	public function setup() {
+
+		// Ensure the plugin is activated.
+		if ( ! defined( 'IA_PLUGIN_VERSION' ) ) {
+			return;
+		}
+
+		if ( ! is_admin() ) {
+			add_filter( 'wp_irving_component_children', [ $this, 'inject_fb_instant_articles_tags_into_head_children' ], 10, 3 );
+		}
+	}
+
+	/**
+	 * Parse the head markup output by FB Instant Articles, injecting it into
+	 * the Head component.
+	 *
+	 * @param array  $children Children for this component.
+	 * @param array  $config   Config for this component.
+	 * @param string $name     Name of this component.
+	 * @return array
+	 */
+	public function inject_fb_instant_articles_tags_into_head_children( array $children, array $config, string $name ): array {
+
+		if ( ! function_exists( 'inject_url_claiming_meta_tag' ) ) {
+			return $children;
+		}
+
+		// Ony run this action on the `irving/head` in a `site` context.
```

Comment:
Do you want this run in `site` context or `page`? The comment doesn't match the code

Reply:
It was page, then I changed it to site, tested, and changed it back w/o updating the comment. Good catch.